src/applications/vaos/referral-appointments/components/ReferralLayout.jsx

```diff
@@ -32,7 +31,7 @@ function BreadCrumbNav() {
           text={text}
           onClick={e => {
             e.preventDefault();
-            dispatch(routeToPreviousReferralPage(history, currentPage));
```

Comment:
@wullaski Is there a reason for the `dispatch` here? I removed it as it was throwing an error